addon/services/page-title-list.js

```diff
@@ -273,3 +275,5 @@ export default class PageTitleListService extends Service {
     headElement.appendChild(titleEl);
   }
 }
+
+PageTitleListService.reopen(Evented);
```

Comment:
Should probably extend at the top `extends `Service.extend(Evented) {`

Reply:
https://github.com/emberjs/ember.js/blob/ec49c70b541b49c70c6453d909482bf1125a0f52/packages/%40ember/-internals/routing/lib/services/router.ts#L64 also does it like this, by keeping the main class definition with the new syntax, I took the idea from them to keep it as consistent as possible with the way the core code is written. What do ya think?

Reply:
I asked in Discord #help about this pattern - https://discord.com/channels/480462759797063690/483601670685720591/775383395249291345 - no responses yet.

Reply:
Maybe. Once again seeing https://github.com/emberjs/rfcs/blob/master/text/0671-modernize-built-in-components-1.md which deprecates calling `reopen` on certain classes gives me pause whenever I think of reaching for it.

Reply:
Plus if mixins were replaced it would be with something like `extends wrapWithEvented(Service) {`

Reply:
@raido I'll watch out for a reply then, thanks!
@knownasilya Yeah, I mean, both approaches make sense... I guess it's a matter of remembering to change it as soon as we have an alternative

Reply:
Yeah, exactly. Which will probably be easy since we'll get a issue once a deprecation starts annoying users :D